### PR TITLE
fix(TsSidenavTriggerComponent)

### PR DESCRIPTION
**Pull request by yevd-denys**
Don't add overlay for 'basic' panel type

https://github.com/GetTerminus/terminus-oss/issues/556

### CODE REVIEW

File: libs/ui/sidenav/src/lib/sidenav-trigger/sidenav-trigger.component.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
   cdkConnectedOverlayHasBackdrop
   cdkConnectedOverlayBackdropClass="ts-sidenav-panel-backdrop"
   [cdkConnectedOverlayOrigin]="triggerItem"
-  [cdkConnectedOverlayOpen]="panelType !== 'drawer' && isPanelOpen"
+  [cdkConnectedOverlayOpen]="panelType === 'popout' && isPanelOpen"
```
Comment (atlwendy) [inline libs/ui/sidenav/src/lib/sidenav-trigger/sidenav-trigger.component.html:46]: Your change means that the panel type has to be `popout` to trigger the overlay open. Are you sure you mean that?

Comment (yevd-denys) [reply libs/ui/sidenav/src/lib/sidenav-trigger/sidenav-trigger.component.html:46]: @atlwendy As I understand for `panelType = undefined` we should not have overlay.
Please correct me If I am wrong.

image

In template we have 2 declarations of overlay (drawer, popout):
```
[cdkConnectedOverlayOpen]="panelType === 'drawer' && isPanelOpen"
[cdkConnectedOverlayOpen]="panelType === 'popout' && isPanelOpen"
```

`export type TS_SIDENAV_PANEL_TYPES = 'popout'  | 'drawer'  | undefined;`

Comment (atlwendy) [reply libs/ui/sidenav/src/lib/sidenav-trigger/sidenav-trigger.component.html:46]: Okay! I assumed that you checked the demo site and the symptom has been fixed, right?
Also want to make sure down the road, if there're more panel types added, we'd also update this accordingly. It is fine currently.

Comment (yevd-denys) [reply libs/ui/sidenav/src/lib/sidenav-trigger/sidenav-trigger.component.html:46]: I have checked it locally and here: [chromatic](https://5f0ca4e61af3790022cad2fe-ryqmmjvabf.chromatic.com/?path=/story/components-navigation-sidenav--basic)

**Review by atlwendy — APPROVED** [review]

**Pull request merged**